auth-server/README.md

```diff
@@ -0,0 +1,17 @@
+# Pet Clinic Authentication Server
+
+It is a server which provides authentication and authorization services.
+
+## Initialize database structure
+1. Use Gradle to run by `gradlew -p auth-server createDatabase` in the root sub-project folder. It creates user and database.
+(see 'gradle.properties' file for configuration)
+2. Use Gradle to run by `gradlew -p auth-server update` in the root sub-project folder. It adds test users to authorize in system.
+3. Use Gradle to run by `gradlew -p auth-server dropDatabase` in the root sub-project folder to delete database and user.
```

Comment:
Create another section to avoid cases like the next one some one can create and drop database